README.md

```diff
@@ -1,2 +1,16 @@
 # maven-properties-example
-An example project for working with system properties.
+
+Thank you for visting our repo. :clap:
```

Comment:
Need to remove this instead.

Reply:
can you remove this line also and add `An example project for working with system properties.` back in.

Reply:
done!